projects/js-packages/connection/components/connect-screen/layout/style.scss

```diff
@@ -99,18 +100,21 @@
 	}
 
 	&__two-columns {
+		display: flex;
+    	flex-wrap: wrap;
```

Comment:
@fgiannar I know this has already been merged, but there is this veeeery small indentation "issue".
The first "tab" is made with spaces instead of tabs, so it doesn't follow the line here 😄 

Checking it out due to mention in https://github.com/Automattic/jetpack/pull/21813